lib/modules/dosomething/dosomething_signup/dosomething_signup.module

```diff
@@ -508,6 +512,8 @@ function dosomething_signup_entity_insert($entity, $type) {
 
   $account = user_load($entity->uid);
   $node = node_load($entity->nid);
+  $transactionals = $entity->transactionals;
```

Comment:
Line 263 of `dosomething_signups.module` adds `transactionals` value to `$values` submitted to `create_entity()`.